demisto_sdk/commands/common/content/tests/objects/pack_objects/pack_metadata/pack_metadata_test.py

```diff
@@ -390,3 +397,69 @@ def test_load_user_metadata_bad_pack_metadata_file(repo, monkeypatch, caplog):
     pack_1_metadata.load_user_metadata("Pack1", "Pack Number 1", pack_1.path, logger)
 
     assert "Failed loading Pack Number 1 user metadata." in caplog.text
+
+
+my_instance = PackMetadata(
```

Comment:
define it inside the test
do you need to send all of the values?

Reply:
Mooved inside.
It is not necessary for me to have the values, but most of them don't have a default value, and for those that do, it's a "pydentic" field that if I don't use it, I get a tester error.

Reply:
fixed.